docs/source/revision_history.rst

```diff
@@ -3,6 +3,11 @@ Revision history
 
 List of major changes:
 
+- Upcoming
+    * Publish ``mypy`` version 0.550 on PyPI.
+
+      * Running mypy now requires Python 3.4 or higher. However Python 3.2 and 3.3 are still valid for the target of the analysis (i.e. the ``--python-version`` flag).
```

Comment:
Can you break this long line? We don't like lines >= 80 characters in our docs; it makes reading on small screens or in side-by-side diffs cumbersome.

Reply:
Also to my embarrassment we actually *don't* support `--python-version 3.2`. You get
```
mypy: error: argument --python-version: Python 3.2 is not supported (must be 3.3 or higher)
```
However using 3.3 as a target is still supported.

Reply:
Oops! Updated!